charters/README.md

```diff
@@ -0,0 +1,18 @@
+# Charter Guide
+
+Each HCA DCP project must have a charter that specifies its scope, responsibilities, and leadership roles.
+
+The charter process provides a simple and consistent path for new projects to be formed that align with the direction of the HCA DCP roadmap and its priorities.
+
+
+## What the process is
+  - Fork the HumanCellAtlas dcp-community repository
+  - Make a subdirectory named charters/`MYPROJECTNAME`
+  - Copy `charters/charter-template.md` to `charters/MYPROJECTNAME/charter.md`
+  - Fill in the charter with attention to detail
+  - Submit a pull request. Add the "charter-proposed" label
+  - Share a link to the pull request for community review on the HumanCellAtlas **#dcp** slack channel
```

Comment:
@brianraymor do you want to have an RFC/Charter-focused slack channel?

Reply:
I don't really want to increase the number of slack channels. I was curious whether we could eliminate the slack announcement and rely on github notifications.